api/src/main/java/org/openmrs/CohortMembership.java

```diff
@@ -43,8 +47,23 @@ public CohortMembership(Patient patient) {
 		this(patient, new Date());
 	}
 	
-	public boolean isMemberActive() {
-		return this.getStartDate() != null && !getStartDate().after(new Date()) && this.getEndDate() == null;
+	/**
+	 * @should return active memberships as of current date
+	 */
+	public boolean isMembershipActive() {
```

Comment:
personally I would have named these methods "isActive" (because it's already on a CohortMembership object). Not a huge deal.